src/main/resources/assets/immersivepetroleum/manual/en_us/speedboat.txt

```diff
@@ -1,15 +1,12 @@
+Wave Rider
+<&speedboat0>The Motorboat is a direct upgrade to the standard wooden boat. The engine in the motorboat runs off the same fuels as the <link;immersivepetroleum:portablegenerator;§n§oPortable Generator§r>, and gives the Motorboat a significant speed boost compared to paddling. Fuel can be added to the tank with any portable fluid canister, such as a bucket or <link;jerrycan;§n§oJerrycan§r>.<np>
+Fuel is consumed while the engine is active at a constant rate whether the Motorboat is moving backwards or forwards. Holding the jump key will increase the speed of the Motorboat, albeit at the cost of higher fuel consumption and increased turn radius.
 
-<np>
-Fuel is consumed while driving the Motorboat forwards or backwards. Holding the jump key will increase the speed of the boat, albeit at the cost of higher fuel consumption and increased turn radius.
-
-Like other devices, the Motorboat can be modified in the <link;workbench;§o§nEngineer's §o§nWorkbench§r>.
-<np>
+Like other devices, the Motorboat can be modified in the <link;workbench;§o§nEngineer's §o§nWorkbench§r>.<np>
 <&speedboat1>
-The §6Expanded Fuel Tank§r allows the Motorboat to hold twice the amount of fuel.
-<&speedboat2>A set of §6Maneuvering Rudders§r gives the Motorboat the ability to change direction much more quickly, drastically improving handling. With this upgrade, the boat's turning radius is impressively lowered.
-<&speedboat3>Navigation through colder climates becomes a trivial task with the §6Icebreaker Bow§r, which allows the Motorboat to ram through ice. In addition to its mobility benefits, the Bow allows the Motorboat to damage monsters that it runs into.
-<&speedboat4>If you seek to explore beyond this realm, or simply wish to make your Motorboat more resistant to the elements, consider installing a §6Reinforced Hull§r. The steel hull grants both increased health and fire resistance to the boat, allowing for navigation of the Nether's vast lava lakes.
-<&speedboat5>Equipping your Motorboat with a pair of §6Emergency Paddles§r means that running out of fuel is no longer the disaster it once was. If the fuel tank on the boat is empty, you can paddle it like a traditional rowboat.
+The §2Expanded Fuel Tank§r allows the Motorboat to hold twice the amount of fuel.
+<&speedboat2>A set of §2Maneuvering Rudders§r gives the Motorboat the ability to change direction much more quickly, drastically improving handling. With this upgrade, the boat's turning radius is impressively lowered.
+<&speedboat3>Navigation through colder climates becomes a trivial task with the §2Icebreaker Bow§r, which allows the Motorboat to ram through ice. In addition to its mobility benefits, the Bow allows the Motorboat to damage monsters that it runs into.
```

Comment:
- Change "**ram**" to
"**plow**"
Clarity
- Change "the **Bow** allows" to
"the **Icebreaker Bow** allows"
Clarity/consistency